src/main/java/org/breedinginsight/brapps/importer/model/base/Observation.java

```diff
@@ -17,39 +17,67 @@
 
 package org.breedinginsight.brapps.importer.model.base;
 
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.Setter;
+import org.brapi.v2.model.pheno.BrAPIObservation;
 import org.breedinginsight.brapps.importer.model.config.*;
 
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
 @Getter
 @Setter
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
 @ImportFieldMetadata(id="Observation", name="Observation",
         description = "An observation object is data that is collected on a trait for a given object being observed.")
 public class Observation implements BrAPIObject {
 
+    public static DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm:ss");
+
+    private static final String TRAIT_NAME = "traitName";
+    private static final String OBSERVATION_UNIT_NAME = "observationUnitName";
+    private static final String STUDY_NAME = "studyName";
```

Comment:
The constants are only referenced internally in each object currently although the processor code should probably be checking this in which case it would make sense to be public. Since it's only doing a name db lookup no check has to be done but if more options were added you would have to. It could also eliminate constant duplication in a couple cases but might be a little confusing because they may not be used in the class where they are defined.

Reply:
created card https://breedinginsight.atlassian.net/browse/BI-1004?atlOrigin=eyJpIjoiYmFjMDQzNWEyNDlhNDYwNWFkM2MxNWQwN2M3NDliYjUiLCJwIjoiaiJ9